app/utils/transformer/DepartureDataTransformer.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils.transformer
+
+import models.UserAnswers
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendHeaderCarrierProvider
+import utils.transformer.transport.border.{IdentificationNumberTransformer, IdentificationTransformer}
+import utils.transformer.transport.eqipment.{ContainerIdentificationNumberTransformer, SealTransformer, TransportEquipmentTransformer}
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class DepartureDataTransformer @Inject() (
+  identificationTransformer: IdentificationTransformer,
+  identificationNoTransformer: IdentificationNumberTransformer,
+  transportEquipmentTransformer: TransportEquipmentTransformer,
+  containerIdTransformer: ContainerIdentificationNumberTransformer,
+  sealTransformer: SealTransformer
```

Comment:
Just wondering if this is going to become quite large with all the transformers we need to write, what do you think?